docs/tr/docs/index.md

```diff
@@ -297,106 +297,106 @@ def update_item(item_id: int, item: Item):
     * `datetime` objesi.
     * `UUID` objesi.
     * Veritabanı modelleri.
-    * ve daha fazlası...
-* 2 alternatif kullanıcı arayüzü dahil olmak üzere, otomatik interaktif API dokümanu:
+    * ve çok daha fazlası...
+* 2 alternatif kullanıcı arayüzü dahil olmak üzere, otomatik etkileşimli API dokümantasyonu sağlar:
     * Swagger UI.
     * ReDoc.
 
 ---
 
-Az önceki kod örneğine geri dönelim, **FastAPI**'ın yapacaklarına bir bakış atalım:
+Az önceki örneğe geri dönelim, **FastAPI**'ın yapacaklarına bir bakış atalım:
 
-* `item_id`'nin `GET` ve `PUT` talepleri içinde olup olmadığının doğruluğunu kontol edecek.
-* `item_id`'nin tipinin `int` olduğunu `GET` ve `PUT` talepleri içinde olup olmadığının doğruluğunu kontol edecek.
-    * Eğer `GET` ve `PUT` içinde yok ise ve `int` değil ise, sebebini belirten bir hata mesajı gösterecek
-* Opsiyonel bir `q` parametresinin `GET` talebi için (`http://127.0.0.1:8000/items/foo?q=somequery` içinde) olup olmadığını kontrol edecek
+* `item_id`'nin `GET` ve `PUT` istekleri için, yolda olup olmadığının kontol edecek.
+* `item_id`'nin `GET` ve `PUT` istekleri için, tipinin `int` olduğunu doğrulayacak.
+    * Eğer değilse, sebebini belirten bir hata mesajı gösterecek.
+* Opsiyonel bir `q` parametresinin `GET` isteği içinde (`http://127.0.0.1:8000/items/foo?q=somequery` gibi) olup olmadığını kontrol edecek
     * `q` parametresini `= None` ile oluşturduğumuz için, opsiyonel bir parametre olacak.
-    * Eğer `None` olmasa zorunlu bir parametre olacak idi (bu yüzden body'de `PUT` parametresi var).
-* `PUT` talebi için `/items/{item_id}`'nin body'sini, JSON olarak okuyor:
-    * `name` adında bir parametetre olup olmadığını ve var ise onun `str` olup olmadığını kontol ediyor.
-    * `price` adında bir parametetre olup olmadığını ve var ise onun `float` olup olmadığını kontol ediyor.
-    * `is_offer` adında bir parametetre olup olmadığını ve var ise onun `bool` olup olmadığını kontol ediyor.
-    * Bunların hepsini en derin JSON modellerinde bile yapacaktır.
-* Bütün veri tiplerini otomatik olarak JSON'a çeviriyor veya tam tersi.
-* Her şeyi dokümanlayıp, çeşitli yerlerde:
-    * İnteraktif dokümantasyon sistemleri.
-    * Otomatik alıcı kodu üretim sistemlerinde ve çeşitli dillerde.
-* İki ayrı web arayüzüyle direkt olarak interaktif bir dokümantasyon sunuyor.
+    * Eğer `None` olmasa zorunlu bir parametre olacaktı (`PUT` metodunun gövdesinde olduğu gibi).
+* `PUT` isteği için `/items/{item_id}`'nin gövdesini, JSON olarak doğrulayıp okuyacak:
+    * `name` adında zorunlu bir parametre olup olmadığını ve varsa tipinin `str` olup olmadığını kontol edecek.
+    * `price` adında zorunlu bir parametre olup olmadığını ve varsa tipinin `float` olup olmadığını kontol edecek.
+    * `is_offer` adında opsiyonel bir parametre olup olmadığını ve varsa tipinin `float` olup olmadığını kontol edecek.
+    * Bunların hepsi en derin JSON objelerinde bile çalışacak.
+* JSON'dan ve JSON'a dönüşümü otomatik olarak yapacak.
```

Comment:
`JSON'dan ve JSON'a` Bu kısım biraz karmaşık görünüyor.

Reply:
`From and to JSON` çevirisini çok düşündüm ancak aklıma daha iyisi gelmedi, önerin var mı?

Reply:
Birden fazla kişiye bu soruyu soruyorum tam bir cevap alamadım ama tahminler üzerine ilerleyebiliyorum.
x => json ve json => x gibi bir anlam ifade ediyor olabilir.

`Verinin JSON'a dönüşümünü otomatik olarak yapacak.` yazılabilir diye düşünüyorum. @hasansezertasan Ne dersiniz?

Reply:
Burada `Convert from and to JSON automatically.` kullanılmış. `vice verca` gibi bir durum var diyebiliriz ancak `JSON'dan ve JSON'a` da pek içime sinmedi şimdi.

Son önerisini inceledim ancak `vice verca` vurgun yok burada.

Aslında ifade etmeye çalıştığı şey, verinin JSON'a ve JSON verinin Python Objesine dönüşümü...

"Verilerin JSON'a ve JSON verilerin de python objelerine dönüşümü otomatik olarak yapılacak."